{"name": "登录异常，测试数据是：{'accounts': 'yaoyao1', 'pwd': '123456', 'exp': '密码错误'}", "status": "passed", "parameters": [{"name": "args", "value": "{'accounts': 'yaoyao1', 'pwd': '123456', 'exp': '密码错误'}"}], "start": 1599776156301, "stop": 1599776156621, "uuid": "60adf0f7-536f-4f5a-bc92-e02588ef5149", "historyId": "9b1c39a42a17c4eb61b2e9ff0cabd0ab", "testCaseId": "acfd50c72596b11b4392c40d7652bccf", "fullName": "script.test_login.TestLogin#test_login_error", "labels": [{"name": "parentSuite", "value": "script"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "DESKTOP-NM1TAU0"}, {"name": "thread", "value": "223692-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "script.test_login"}]}